scripts/lease.sh

```diff
@@ -20,3 +22,14 @@ delete_lease() {
   k8s-claimer --server=k8s-claimer-e2e.deis.com lease delete $TOKEN
   return $?
 }
+
+get-pod-logs() {
+  local components="deis-builder deis-database deis-minio deis-registry deis-router deis-controller"
```

Comment:
shall we add one or more of `deis-workflow-manager deis-logger deis-monitor-grafana deis-monitor-influxapi deis-monitor-influxui deis-monitor-stdout`?  if all, can we use a wildcard here?  `deis-*` perhaps?